Caltech-IPAC/firefly
# Firefly-1413: Implement new button menu

#### [Firefly-1413](https://jira.ipac.caltech.edu/browse/FIREFLY-1413): Implement new button menu design

   - see ticket for details
   - not done: landing page still needs some work

irsa-ife PR: https://github.com/IPAC-SW/irsa-ife/pull/307

#### Testing
- URLs
  - https://firefly-1413-blue-buttons.irsakudev.ipac.caltech.edu/irsaviewer
  - https://firefly-1413-blue-buttons.irsakudev.ipac.caltech.edu/irsaviewer/dce
  - https://firefly-1413-blue-buttons.irsakudev.ipac.caltech.edu/irsaviewer/timeseries
  - https://fireflydev.ipac.caltech.edu/firefly-1413-blue-buttons/firefly
  - https://fireflydev.ipac.caltech.edu/firefly-1413-blue-buttons/firefly/ts.html
- be aware you menu is saved as a preference so when you reload it it will be the same
- Things to try:
      - navigate with the side bar
      - change tri-view layout with the side bar
      - notice the dce and timeseries do not default the the result tab
- Things to note about IrsaViewer. I used it to demonstrate how we can have more TAP menu items locked to a service. This is not permeant, it is just to help us understand what is posible

Todo:

- [x] timeseries should support drag-and-drop
- [x] tab button text should not wrap when not enough room
- [x] image panel should not lock to coverage-replace-mode any more
- [x] Support TAP obscore lock
- [x] add icon to upload to make it have a more aligned place.
- [x] make side bar header a little smaller.
- [x] make tab/button resize smoother
